sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlobBaseApiTests.java

```diff
@@ -521,8 +522,7 @@ public void queryFatalError() {
 
         liveTestScenarioWithRetry(() -> {
             /* Input Stream. */
-            InputStream qqStream = bc.openQueryInputStreamWithResponse(options).getValue();
-            assertThrows(Throwable.class, () -> readFromInputStream(qqStream, Constants.KB));
```

Comment:
Is there a particular reason these tests were updated? Specifically, it looks like this logic throws an error when the input stream is first opened instead of on reads? I noticed we were doing this as well in the `datalake/FileApiTest.java` test as well.